src/main/java/com/google/firebase/auth/OidcProviderConfig.java

```diff
@@ -116,9 +123,9 @@ CreateRequest getThis() {
      * {@link AbstractFirebaseAuth#updateOidcProviderConfig(CreateRequest)} to update the provider
      * information persistently.
      *
-     * @param tenantId a non-null, non-empty provider ID string.
-     * @throws IllegalArgumentException If the provider ID is null or empty, or if it's an invalid
-     *     format
+     * @param tenantId A non-null, non-empty provider ID string.
+     * @throws IllegalArgumentException If the provider ID is null or empty, or if the format is
+     *     invalid.
```

Comment:
Nit: Perhaps specify the prefix (oidc.*) requirement

Reply:
Good idea! Done.